messaging/handler.go

```diff
@@ -231,7 +231,9 @@ func CopyFlush(dst io.Writer, src io.Reader) (written int64, err error) {
 			}
 
 			// Flush after write.
-			if dst, ok := dst.(http.Flusher); ok {
```

Comment:
This one wasn't necessary. I mostly just don't like using http.Flusher because Flush() is such a generic concept that tying it to http seems dumb. There is no functional change here. It's just aesthetics.